src/core/toga/widgets/numberinput.py

```diff
@@ -80,8 +89,8 @@ def min_value(self, value):
         except ValueError:
             raise ValueError("min_value must be an integer")
         except TypeError:
-            self._min_value = None
-        self._impl.set_min_value(value)
+            self._min_value = int(-1.7976931348623157e+308)
```

Comment:
So, this float constant exists as `sys.float_info.max`, so it would be preferable to use that constant rather than redefine it manually.

However, I'm not sure it's the best constant to use here. Although Python integers can be arbitrarily large, in Toga they're going to be backed by widgets that are implemented in C... which means there *will* be a practical max/min value. 

I'd suggest using `sys.max_size` - it's an integer value, equivalent to 2**63-1, internally described as the largest practical index for an array or string.